src/python/pants/backend/python/rules/coverage_integration_test.py

```diff
@@ -10,139 +10,139 @@
-                    )
-
-                    python_tests(
-                      name="arithmetic",
-                      sources=["test_arithmetic.py"],
-                      dependencies=['{tmpdir_relative}/src/python/project'],
-                    )
-                    """
+        )
+        (test_root / "BUILD").write_text(
+            dedent(
+                f"""\
+                python_tests(
+                    name="multiply",
+                    sources=["test_multiply.py"],
+                    dependencies=['{tmpdir_relative}/src/python/project'],
                 )
-            )
 
-            # Test a file that does not cover any src code. While this is unlikely to happen, this
-            # tests that we can properly handle the edge case. In particular, we want to make sure
-            # that coverage still works when we omit this test file through the option
-            # `--omit-test-sources`.
-            no_src_folder = Path(tmpdir, "tests", "python", "project_test", "no_src")
-            no_src_folder.mkdir()
-            (no_src_folder / "__init__.py").touch()
-            (no_src_folder / "test_no_src.py").write_text(
-                "def test_true():\n\tassert True is True\n"
-            )
-            (no_src_folder / "BUILD").write_text("python_tests()")
-
-            command = [
-                "--no-v1",
-                "--v2",
-                "test",
-                "--use-coverage",
-                f"{tmpdir_relative}/src/python/project:tests",
-                f"{tmpdir_relative}/tests/python/project_test:multiply",
-                f"{tmpdir_relative}/tests/python/project_test:arithmetic",
-                f"{tmpdir_relative}/tests/python/project_test/no_src",
-            ]
-            default_result = self.run_pants(command)
-            filter_result = self.run_pants(
-                [*command, "--coverage-py-filter=['project.lib', 'project_test.no_src']"]
+                python_tests(
+                    name="arithmetic",
+                    sources=["test_arithmetic.py"],
+                    dependencies=['{tmpdir_relative}/src/python/project'],
+                )
+                """
             )
+        )
+        # Test a file that does not cover any src code. While this is unlikely to happen, this
+        # tests that we can properly handle the edge case. In particular, we want to make sure
+        # that coverage still works when we omit this test file through the option
+        # `--omit-test-sources`.
+        no_src_folder = Path(tmpdir, "tests", "python", "project_test", "no_src")
+        no_src_folder.mkdir()
+        (no_src_folder / "__init__.py").touch()
+        (no_src_folder / "test_no_src.py").write_text("def test_true():\n\tassert True is True\n")
+        (no_src_folder / "BUILD").write_text("python_tests()")
+        return tmpdir_relative
+
+    def _run_tests(self, tmpdir_relative, *more_args: str):
```

Comment:
This is from `pants_run_integration_test.py`. Also fine to fix in a followup.

```suggestion
    def _run_tests(self, tmpdir_relative, *more_args: str) -> PantsResult:
```